MediaBrowser.Model/Drawing/ImageResolution.cs

```diff
@@ -0,0 +1,53 @@
+namespace MediaBrowser.Model.Drawing
```

Comment:
New files should use [file scoped namespaces](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/file-scoped-namespaces)